src/main/java/trello/entities/Context.java

```diff
@@ -67,4 +80,31 @@ public Board getBoard() {
     public List getList() {
         return list;
     }
+
+    /**
+     * Sets context's user.
+     *
+     * @param list is the user to be shared in context.
+     */
+    public void setList(final List list) {
+        this.list = list;
+    }
+
+    /**
+     * Gets the context's card.
+     *
+     * @return the card that is on the context.
+     */
+    public Card getCard() {
+        return card;
+    }
+
+    /**
+     * Sets context's user.
```

Comment:
Please fix this comment you set a Card not User.

Reply:
Solved